### DISCUSSION TITLE
How to Set Different Page Titles and Meta Titles in MASA CMS Similar to Yoast SEO in WordPress?

### DISCUSSION THREAD

**Discussion by laxman-coozmoo** [MasaCMS/MasaCMS#340] [Q&A]
Hello Masa , I would like to know if it's possible to have different titles for the page and meta title in MASA CMS. For example, in WordPress, we can use the Yoast SEO plugin to set a different meta title while keeping the page title separate. Is there any similar plugin or method available in MASA CMS to achieve this? If so, could you please guide me on how to fix this?
![Screenshot 2024-10-24 194033](https://github.com/user-attachments/assets/4677c1b6-25e6-47ac-9da7-4b1c77cbdb73)
![Screenshot 2024-10-24 193953](https://github.com/user-attachments/assets/7c2d1ec6-ed51-486a-b12b-5549cb67add3)

**Comment by Tigerlady** [comment]
Yes, there is a way to do this without a plugin.  Under the Basic tab, there are several fields that you can use for different titles
![image](https://github.com/user-attachments/assets/739ad38b-1bc1-4e85-b97d-ddf05d582e97)

Also, under the Summary Tab, you could possibly use one of these fields and just call in your code.

**Reply by laxman-coozmoo** [reply]
Thanks for the response, but I've already tried making those changes. However, the page title and meta title are not showing any difference. If I leave the title blank, the page won't save because it's a required field. The "HTML (Browser) Title (Optional)" option is prioritizing the title over both the page title and meta title.

**Comment by Tigerlady** [comment]
Do you have access to the code of your site, Specifically the "html_head.cfm" page? If so, you can change the markup on the page to use whichever field you want.  See below; you can use whatever field you want for each section.  For example, you could change the HTML Title to whatever field you want.  Also, you could create an extended attribute in the page class extension that you could use in place of whatever you want. 

`<meta name="description" content="#encodeForHTML($.content('metaDesc'))#">
	<meta name="keywords" content="#encodeForHTML($.content('metaKeywords'))#">
	

	<title>#esapiEncode('html', $.content('HTMLTitle'))# - #esapiEncode('html', $.siteConfig('site'))#</title>`
